<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BolusScreen</class>
 <widget class="QWidget" name="BolusScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="carbs_textEdit">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>carbs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="bg_textEdit">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>BG</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delivery:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Immediate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="text">
      <string>Delayed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>24</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calculate_pushButton">
     <property name="text">
      <string>calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Calculation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser_2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calculate_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>BolusScreen</receiver>
   <slot>calculateBolus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculateBolus()</slot>
  <slot>setExtendedHours()</slot>
 </slots>
</ui>
